tests/python/unittest/test_io.py

```diff
@@ -294,7 +294,7 @@ def test_DataBatch():
 
 @unittest.skip("test fails intermittently. temporarily disabled till it gets fixed. tracked at https://github.com/apache/incubator-mxnet/issues/7826")
```

Comment:
seems like this test is being skipped . is the issue fixed now ?

Reply:
I looked at the issue mentioned in the skip message, it was some issue on Windows. I've manually run this test on my Ubuntu machine and this test passes for both dtypes. On the other hand this PR has already been tested on real workloads and proven to work.